src/components/Breadcrumbs/README.md

```diff
@@ -1,43 +1,345 @@
+            {
+                text: 'District',
+                action: () => {},
+            },
+            {
+                text: 'Street',
+                action: () => {},
+            },
+        ]}
+        firstDisplayedItemsCount={1}
+        lastDisplayedItemsCount={1}
+    />
+</ExampleBlock>
+
+LANDING_BLOCK-->
 
-### Examples
+<!--GITHUB_BLOCK-->
 
 ```jsx
 const breadcrumbs = [
   {
-    title: 'What is love',
+    text: 'Region',
+    action: () => {},
+  },
+  {
+    text: 'Country',
+    action: () => {},
   },
   {
-    title: "Baby don't hurt me",
+    text: 'City',
+    action: () => {},
   },
   {
-    title: "Don't hurt me",
+    text: 'District',
+    action: () => {},
   },
   {
-    title: 'No more',
+    text: 'Street',
+    action: () => {},
   },
 ];
 
 return (
   <Breadcrumbs
     items={items}
+    firstDisplayedItemsCount={FirstDisplayedItemsCount.One}
     lastDisplayedItemsCount={LastDisplayedItemsCount.One}
+  />
+);
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Custom divider
+
+<!--LANDING_BLOCK
+
+<ExampleBlock
+    code={`
+<Breadcrumbs
+    items={[
+        {
+            text: 'Region',
+            action: () => {},
+        },
+        {
+            text: 'Country',
+            action: () => {},
+        },
+        {
+            text: 'City',
+            action: () => {},
+        },
+        {
+            text: 'District',
+            action: () => {},
+        },
+        {
+            text: 'Street',
+            action: () => {},
+        },
+    ]}
+    renderItemDivider={() => '>'}
     firstDisplayedItemsCount={FirstDisplayedItemsCount.One}
+    lastDisplayedItemsCount={LastDisplayedItemsCount.One}
+/>
+`}
+>
+    <UIKit.Breadcrumbs
+        items={[
+            {
+                text: 'Region',
+                action: () => {},
+            },
+            {
+                text: 'Country',
+                action: () => {},
+            },
+            {
+                text: 'City',
+                action: () => {},
+            },
+            {
+                text: 'District',
+                action: () => {},
+            },
+            {
+                text: 'Street',
+                action: () => {},
+            },
+        ]}
+        renderItemDivider={() => '>'}
+        firstDisplayedItemsCount={1}
+        lastDisplayedItemsCount={1}
+    />
+</ExampleBlock>
+
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```jsx
+const breadcrumbs = [
+  {
+    text: 'Region',
+    action: () => {},
+  },
+  {
+    text: 'Country',
+    action: () => {},
+  },
+  {
+    text: 'City',
+    action: () => {},
+  },
+  {
+    text: 'District',
+    action: () => {},
+  },
+  {
+    text: 'Street',
+    action: () => {},
+  },
+];
+
+return (
+  <Breadcrumbs
+    items={items}
+    renderItemDivider={() => '>'}
+    firstDisplayedItemsCount={FirstDisplayedItemsCount.One}
+    lastDisplayedItemsCount={LastDisplayedItemsCount.One}
   />
 );
 ```
+
+<!--/GITHUB_BLOCK-->
+
+### Custom title
```

Comment:
Seems like a part of `BreadcrumbsItem` item menu, but not the part of the Appearance